tasks/prime_hhg_workflow.py

```diff
@@ -131,64 +135,69 @@ def get_stored(self, object_key, object_id=None):
 
     def add_stored(self, object_key, object_data):
         """Adds the object to the main move."""
-
-        # If it's a moveTaskOrder, we only keep track of one move per workflow, so add actually replaces.
         if object_key == MOVE_TASK_ORDER:
-            self.current_move = object_data
-
-        # If it's the following object types, we add them into nested array in the MTO
-        elif object_key in [MTO_SHIPMENT, MTO_SERVICE_ITEM, PAYMENT_REQUEST]:
-            if isinstance(object_data, list):
-                self._add_nested_array_to_mto(object_key, object_data)
-            else:
-                self._add_nested_object_to_mto(object_key, object_data)
+            self.current_move_id = object_data["id"]
+
+        # # If it's a moveTaskOrder, we only keep track of one move per workflow, so add actually replaces.
+        # if object_key == MOVE_TASK_ORDER:
+        #     self.current_move = object_data
+        #
+        # # If it's the following object types, we add them into nested array in the MTO
+        # elif object_key in [MTO_SHIPMENT, MTO_SERVICE_ITEM, PAYMENT_REQUEST]:
+        #     if isinstance(object_data, list):
+        #         self._add_nested_array_to_mto(object_key, object_data)
+        #     else:
+        #         self._add_nested_object_to_mto(object_key, object_data)
 
     def update_stored(self, object_key, old_data, new_data):
         """Replaces the object in old_data with the one in new_data"""
-        if object_key == MOVE_TASK_ORDER:
-            if self.current_move["id"] == old_data["id"]:
-                self.current_move = new_data
-            else:
-                raise ImplementationError(
-                    f"Cannot update move with object because its ID of {old_data['id']} does not "
-                    f"match the current move's id of {self.current_move['id']}."
-                )
-        elif object_key in [MTO_SHIPMENT, MTO_SERVICE_ITEM, PAYMENT_REQUEST]:
-            self._update_nested_object_in_mto(object_key, old_data, new_data)
+        pass
```

Comment:
We no longer need to do this since we are pull the move id and requested the original move if we need an mto now.